SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -553,6 +607,18 @@ void USpatialSender::SendDeleteEntityRequest(Worker_EntityId EntityId)
 	Connection->SendDeleteEntityRequest(EntityId);
 }
 
+void USpatialSender::SendClearRPCsOnEntityCreationRequest(Worker_EntityId EntityId)
+{
+	Worker_CommandRequest CommandRequest = RPCsOnEntityCreation::CreateClearFieldsCommandRequest();
+	NetDriver->Connection->SendCommandRequest(EntityId, &CommandRequest, SpatialConstants::CLEAR_RPCS_ON_ENTITY_CREATION);
+}
+
+void USpatialSender::SendRPCsOnEntityCreationComponentUpdate(Worker_EntityId EntityId)
```

Comment:
I think these two function names can be a bit more clear regarding what they are doing. Potentially
* `SendRequestToClearRPCsOnEntityCreation` and then maybe have a `check` inside the function making sure it's only ever sent from a client.
* `ClearRPCsOnEntityCreation` and have a check it's on the server.

Reply:
Renamed, added server check.